pages/common/composer.md

```diff
@@ -0,0 +1,23 @@
+# composer
+
+> A per-project package manager for PHP.
+
+- Update composer to the latest version:
+
+`composer self-update`
+
+- Make a package a dependency for this project:
```

Comment:
Yeah, it's very similar to npm's `package.json`. It installs things local to the folder you're in (current directory) - except it installs to `vendor`, whereas npm installs to `node_modules`.